Source/Eto.WinForms/Forms/Controls/DrawableHandler.cs

```diff
@@ -29,6 +29,27 @@ public EtoDrawable ()
 				this.SetStyle (swf.ControlStyles.SupportsTransparentBackColor, true);
 			}
 
+			public override sd.Rectangle DisplayRectangle
+			{
+				get
+				{
+					var rect = ClientRectangle;//base.DisplayRectangle;
+					var padding = Padding;
+					rect.X += padding.Left;
+					rect.Y += padding.Top;
+					rect.Width -= padding.Horizontal;
+					rect.Height -= padding.Vertical;
+					return rect;
+				}
+			}
```

Comment:
this is what `ScrollableControl` does and what `DefaultLayout` requires to properly propose size.